DFReader.py

```diff
@@ -59,7 +65,8 @@ def __init__(self, type, name, flen, format, columns):
                 msg_mults.append(mul)
                 msg_types.append(type)
             except KeyError as e:
-                raise Exception("Unsupported format char: '%s' in message %s" % (c, name))
```

Comment:
I am OK with long lines it that is the way to go.

I too have great worries about unqualified exceptions... https://realpython.com/blog/python/the-most-diabolical-python-antipattern/